flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/TestingDispatcher.java

```diff
@@ -159,12 +160,14 @@ void completeJobExecution(ExecutionGraphInfo executionGraphInfo) {
     }
 
     CompletableFuture<Void> getJobTerminationFuture(@Nonnull JobID jobId, @Nonnull Time timeout) {
```

Comment:
What about using `Duration` here and in `getNumberJobs` as well?

Reply:
Eventually. I want to go through things component-by-component, and this belongs to the dispatcher.